package.json

```diff
@@ -4,8 +4,14 @@
   "version": "0.1.5-beta",
   "author": "Teamleader <development@teamleader.eu>",
   "betterScripts": {
+    "storybook": {
+      "command": "start-storybook -p 3000",
```

Comment:
Q: can you make it possible to change the port? I run this on a different port most of the times.